src/App.css

```diff
@@ -268,6 +269,16 @@ span {
 	width: 100%;
 }
 
+.hover-btn{
+    transition:background-color 2s ease;
+}
+
+.hover-btn:hover{
+    transform: scale(1.03);
+    /* display: none; */
+    background-color: var(--primary-color) !important;
```

Comment:
Prefer not using important. Use only if there's no other option left

Reply:
okay will try this